### DISCUSSION TITLE
Non Root Docker Image

### DISCUSSION THREAD

**Discussion by J12934** [directus/directus#4577] [Archive]
Hi 👋
We've noticed that the default directus docker images (e.g. `directus/directus:v9.0.0-rc.51`) run with a root user , this is generally considered a bad practice and hinders us from running it in our kubernetes cluster, as non root users are enforced by a PodSecurityPolicy in our and many other clusters.

Is there a specific reason for having directus run as root inside the image?

I'd be willing to open up a PR which adds a non root user to image.

**Comment by benhaynes** [comment]
@WoLfulus ?

**Comment by paescuj** [comment]
For future visitors of this thread: This has been addressed quite some time ago in [v9.0.0-rc.71](https://github.com/directus/directus/releases/tag/v9.0.0-rc.71) via https://github.com/directus/directus/pull/5877.

**Reply by requiem240sx** [reply]
I noticed this after I posted, you are correct! Thanks, I removed my original comment!
